infra/templates/az-isolated-service-single-region/outputs.tf

```diff
@@ -1,6 +1,39 @@
+// ---- Admin Resources ----
+
+output "admin_resource_group" {
+  value = azurerm_resource_group.admin_rg.name
+}
+
 output "fqdns" {
   value = [
-      for uri in module.app_service.app_service_uri:
-        "http://${uri}"
+    for uri in module.app_service.app_service_uri :
+    "http://${uri}"
   ]
+}
+
+output "app_insights_id" {
+  value = module.app_insights.app_insights_app_id
+}
+
+output "service_plan_name" {
+  value = module.service_plan.service_plan_name
+}
+
+output "service_plan_id" {
+  value = module.service_plan.app_service_plan_id
+}
+
+
+// ---- App Dev Resources ----
+
+output "app_dev_resource_group" {
+  value = azurerm_resource_group.app_rg.name
+}
+
+output "keyvault_name" {
+  value = module.keyvault.keyvault_name
+}
+
+output "keyvault_id" {
+  value = module.keyvault.keyvault_id
```

Comment:
Why do we need to surface this as an output?

Reply:
I thought i'd need it to access the keyvault resource when writing my tests but I think its actually not needed. I'll remove it.